lib/msf/core/exploit.rb

```diff
@@ -750,6 +750,67 @@ def compatible_payloads(excluded_platforms: [], excluded_archs: [])
     payloads
   end
 
+  # Select a reasonable default payload and minimally configure it
+  # @param [Msf::Module] mod
+  def self.choose_payload(mod)
```

Comment:
Let's move this over to the Payload class

Reply:
Got that moved 👍